falkordb/falkordb.py

```
@@ -25,88 +26,121 @@ class FalkorDB():
+            health_check_interval=health_check_interval,
+            client_name=client_name,
+            lib_name=lib_name,
+            lib_version=lib_version,
+            username=username,
+            retry=retry,
+            redis_connect_func=connect_func,
+            credential_provider=credential_provider,
+            protocol=protocol,
+        )
 
         if Is_Sentinel(conn):
```

Ruff

`Is_Sentinel` may be undefined, or defined from star imports

(F405)